public/js/forecast.js

```diff
@@ -0,0 +1,13 @@
+const forecastGetter = async (location) => {
```

Comment:
Functions and methods names must be verbs.

Reply:
Fixed.